### PR TITLE
Completed Hibernate-01

### PR DESCRIPTION
**Pull request by Vedroid**

### CODE REVIEW

File: src/main/resources/hibernate.cfg.xml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<?xml version = "1.0" encoding = "utf-8"?>
+<!DOCTYPE hibernate-configuration SYSTEM
+        "http://www.hibernate.org/dtd/hibernate-configuration-3.0.dtd">
+<hibernate-configuration>
+    <session-factory>
+        <property name="hibernate.dialect">org.hibernate.dialect.PostgreSQL10Dialect</property>
+        <property name="hibernate.connection.driver_class">org.postgresql.Driver</property>
+        <!-- currentSchema=cinema& -->
```
Comment (Max-Pochepets) [inline src/main/resources/hibernate.cfg.xml]: no need

**Review by Max-Pochepets — APPROVED** [review]
👍

File: src/main/java/ua/vedroid/cinema/dao/impl/MovieDaoImpl.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+            session = HibernateUtil.getSessionFactory().openSession();
+            transaction = session.beginTransaction();
+            session.save(movie);
+            transaction.commit();
+            return movie;
+        } catch (Exception e) {
+            if (transaction != null) {
+                transaction.rollback();
+            }
+            throw new DataProcessingException("Can`t insert Movie entity", e);
```
Comment (Sofasmile) [inline src/main/java/ua/vedroid/cinema/dao/impl/MovieDaoImpl.java]:
```suggestion
            throw new DataProcessingException("Can`t insert Movie entity " + movie, e);
```

**Review by Sofasmile — CHANGES_REQUESTED** [review]
good job)

**Review by Sofasmile — APPROVED** [review]

**Pull request merged**